webpack.config.js

```diff
@@ -5,7 +5,11 @@ Encore
     .setPublicPath('/build')
     .cleanupOutputBeforeBuild()
     .addStyleEntry('global', './public/scss/application.scss')
-    .addLoader({ test: /\.scss$/, loader: 'import-glob-loader' })
+    .addLoader({test: /\.scss$/, loader: 'webpack-import-glob-loader'})
+    .configureLoaderRule('eslint', loaderRule => {
+        loaderRule.test = /\.(jsx?|vue)$/
```

Comment:
What does this do?

Reply:
Thanks for pointing this out. 

To answer your question. This loads eslint and configures it to mark jsx and vue files a test resources

And that is completely out of scope for this project. Removed this loader.

FYI This code came from another stepup project where we already upgraded to Encore 4. So I yanked some of the updated config from there. But kept a bit too much :blush: